src/middlewared/middlewared/plugins/boot_/boot_loader_freebsd.py

```diff
@@ -0,0 +1,26 @@
+import os
+import tempfile
+
+from middlewared.service import Service
+from middlewared.utils import run
+
+from .boot_loader_base import BootLoaderBase
+
+
+class BootService(Service, BootLoaderBase):
+
+    async def install_loader(self, dev):
+        if (await self.middleware.call('boot.get_boot_type')) == 'EFI':
+            with tempfile.TemporaryDirectory() as tmpdirname:
+                await run('mount', '-t', 'msdosfs', f'/dev/{dev}p1', tmpdirname, check=False)
+                try:
+                    os.makedirs(f'{tmpdirname}/efi/boot')
```

Comment:
I think there is a exist_ok=True arg to that method

Reply:
Yeah there is and i actually had it in my mind i'll be covering it. Forgot :( Thank you for catching it :)